OctopusDSC/Tests/cOctopusServer.Tests.ps1

```diff
@@ -20,130 +20,265 @@ try
             function Get-Service {}
         }
 
-        function Get-DesiredConfiguration {
-            $pass = ConvertTo-SecureString "S3cur3P4ssphraseHere!" -AsPlainText -Force
-            $cred = New-Object System.Management.Automation.PSCredential ("Admin", $pass)
-
-            [System.Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseDeclaredVarsMoreThanAssignments', '')]
-            $desiredConfiguration = @{
-                 Name                   = 'Stub'
-                 Ensure                 = 'Present'
-                 State                  = 'Started'
-                 WebListenPrefix        = "http://localhost:80"
-                 SqlDbConnectionString  = "conn-string"
-                 OctopusAdminCredential = $cred
-            }
-            return $desiredConfiguration
-        }
-
         Describe 'cOctopusServer' {
-            BeforeEach {
-
-            }
-
             [System.Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseDeclaredVarsMoreThanAssignments', '')]
             $mockConfig = [pscustomobject] @{
                 OctopusStorageExternalDatabaseConnectionString         = 'StubConnectionString'
                 OctopusWebPortalListenPrefixes                         = "https://octopus.example.com,http://localhost"
             }
             Mock Import-ServerConfig { return $mockConfig }
 
-            Context 'Get-TargetResource' {
-                Mock Get-ItemProperty { return @{ InstallLocation = "c:\Octopus\Octopus" }}
-                Mock Get-Service { return @{ Status = "Running" }}
+            function Get-DesiredConfiguration {
+                $pass = ConvertTo-SecureString "S3cur3P4ssphraseHere!" -AsPlainText -Force
+                $cred = New-Object System.Management.Automation.PSCredential ("Admin", $pass)
+
+                [System.Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseDeclaredVarsMoreThanAssignments', '')]
+                $desiredConfiguration = @{
+                     Name                   = 'Stub'
+                     Ensure                 = 'Present'
+                     State                  = 'Started'
+                     WebListenPrefix        = "http://localhost:80"
+                     SqlDbConnectionString  = "conn-string"
+                     OctopusAdminCredential = $cred
+                }
+                return $desiredConfiguration
+            }
 
-                It 'Returns the proper data' {
```

Comment:
This test had been duplicated by `'When reg key exists and service exists and service started'`